pdfannots/__init__.py

```diff
@@ -71,6 +71,16 @@ def _mkannotation(
         # decode as string, normalise line endings, replace special characters
         contents = cleanup_text(pdfminer.utils.decode_text(contents))
 
+    color = pa.get('C')
```

Comment:
Does every annotation have a 'C' attribute? I suspect this may legitimately be None (e.g. for note annotations), in which case we probably shouldn't emit the warning on line 80. I would only try to parse the colour and print the warning if C is set but invalid.